main.go

```diff
@@ -78,7 +79,9 @@ func (hc *HealthCheckWebService) apiGetIsReplica(w http.ResponseWriter, r *http.
 }
 
 func main() {
-	cfg, err := config.ParseConfig("./examples/config.yml")
+	pathToConfig := os.Args[1]
```

Comment:
@film42 I noticed you used cobra in pghost. Are you cool with accessing command line arguments via `os.Args`? Or should we use a library like cobra?

Reply:
@benschinn what happens if someone doesn't pass in an argument?

Reply:
I was expecting an exception to be thrown here:
https://github.com/film42/pgreba/blob/master/config/config.go#L35

but it turns out that's not the case. I will do a check on the args length, and throw an exception if the length is less than 2.

Reply:
https://github.com/film42/pgreba/pull/7/commits/88f07ab3b1719ea573c487b9c93311611f41e11f